Documentation/Getting-Started/Prerequisites/prerequisites.md

```diff
@@ -7,7 +7,7 @@ and Rook is granted the required privileges (see below for more information).
 
 ## Minimum Version
 
-Kubernetes **v1.21** or higher is supported.
+Kubernetes **v1.22** or higher is supported.
```

Comment:
```suggestion
Kubernetes **v1.22.17** or higher is supported.
```

can we be more specific?

Reply:
Sure, changed at other places in Doc too.